test/verify/check-machines

```diff
@@ -392,7 +392,16 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         b.wait_in_text(".memory-usage-chart .pf-c-progress__status > .pf-c-progress__measure", "0 /")
         b.wait_in_text(".vcpu-usage-chart .pf-c-progress__status > .pf-c-progress__measure", "0%" )
 
+        # shut off of a transient VM will redirect us to the main page
         self.goToMainPage()
+        m.execute("virsh dumpxml subVmTest1 > /tmp/subVmTest1.xml")
+        m.execute("virsh start {0} && virsh undefine {0}".format("subVmTest1"))
+        b.wait_visible("tr[data-row-id=vm-subVmTest1-system][data-row-transient=true]")
+        self.goToVmPage("subVmTest1")
+        b.click("#vm-subVmTest1-action-kebab button")
+        b.click("#vm-subVmTest1-forceOff")
+        b.wait_not_present("#vm-details")
```

Comment:
This is a bit sloppy -- it would succeed if the page just stopped rendering anything, but it makes no assertion about that you really end up at the VM overview. Could you add a `wait_visible()` for the overview here?

If the page was really broken, then I figure the subsequent `b.click("#vm-subVmTest1-action-kebab button")` may fail, but such a clone action *might* also exist on the details page. So I think it's both less confusing and also more robust to explicitly say what we expect -- the "redirect us to the main page".